src/baldr/accessrestriction.cc

```diff
@@ -1,7 +1,8 @@
 #include "baldr/accessrestriction.h"
 #include "baldr/timedomain.h"
+#include "tyr/serializers.h"
```

Comment:
this is a serious line we dont want to cross... it makes tyr a dependency of pretty much everything (except midgard) if we do this. why is this include even needed? all the this file should need is the rapidjson util header